Marlin/src/lcd/extui/lib/mks_ui/draw_tool.cpp

```diff
@@ -71,7 +71,9 @@ static void event_handler(lv_obj_t *obj, lv_event_t event) {
       uiCfg.desireSprayerTempBak = thermalManager.temp_hotend[uiCfg.curSprayerChoose].target;
       lv_draw_filament_change();
       break;
-    case ID_T_MORE: lv_draw_more(); break;
+      case ID_T_MORE:
+        TERN_(CUSTOM_USER_MENUS, lv_draw_more());
```

Comment:
Because G-Code "console" is always enabled, I think this TERN is not required. Either the more menu will only have G-Code console, either it'll have G-Code console + what user enabled. This *might* allow to remove the `CUSTOM_USER_MENUS` define also?

Reply:
I will remove the if